.idea/.name

```diff
@@ -0,0 +1 @@
+30-seconds-of-php-code
```

Comment:
Do you follow the [CONTRIBUTING](https://github.com/appzcoder/30-seconds-of-php-code/blob/master/CONTRIBUTING.md) guide?
This file is not necessary.